docs/Support/FAQ.md

```diff
@@ -515,8 +520,73 @@ To retrieve data from a REST endpoint using a service task (`serviceTaskOperator
 **Q:** Why do I receive different behaviors when running the same service task in SpiffWorkflow library vs. SpiffArena?
 
 **A:**
-In SpiffWorkflow library, service tasks do not have built-in functionality and are essentially placeholders that require external implementations to function. In contrast, SpiffArena requires connectors to be set up for service tasks to function, which might lead to errors if the connectors or their configurations are incorrect. Errors such as "invalid syntax" typically occur when expressions (like URLs) are not properly formatted.
+In SpiffWorkflow library, service tasks do not have built-in functionality and are essentially placeholders that require external implementations to function. 
 
-### **45: not JSON serializable error in Script Task**
+In contrast, SpiffArena requires connectors to be set up for service tasks to function, which might lead to errors if the connectors or their configurations are incorrect. Errors such as "invalid syntax" typically occur when expressions (like URLs) are not properly formatted.
 
-Please see [documentation on what is possible in Script Tasks](/Building_Diagrams/Script_Tasks.md#what-can-you-do-and-not-do-in-script-tasks).
+**Q:** How do I configure and fix errors in HTTP Service Tasks?**
+
+**A:**
+Common errors occur when configuring HTTP Service Tasks.
+- **Configuration Basics:**
+  - **Headers:** Set to `{}` if no headers are required.
+  - **Basic Authentication:** Use `basic_auth_username` and `basic_auth_password` fields.
+  - **Data Payload:** Provide a JSON object in the `data` field for POST requests.
+- **Common Errors:**
+  - **Missing `headers`:** Add `{}` to the `headers` field.
+  - **Connection Refused:** Replace `localhost` with `host.docker.internal` in containerized environments.
+
+### **45: Mapping input/Output for Call Activity Subprocess**
+**Q:**  How do I map input and output for Call Activity subprocesses?**
+
+**A:** In the subprocess:
+  - Use single `=` for variable assignments (e.g., `result = "approve"`).
+  - Avoid using double `==`, as it is a comparison operator, not an assignment.
+  
+  Also, ensure input/output mappings are correctly defined in the parent process to facilitate data transfer.
+
+### **46: Embed the BPMN editor in an iframe**
```

Comment:
I don’t think we’ve ever successfully used the editor in an iframe. I’d probably avoid this question to avoid misleading people.